### PR TITLE
Zib-AbilityToUseToilet

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**

### CODE REVIEW

File: known-issues.yml [hunk 1]
```diff
@@ -67,7 +73,7 @@ zib-ContactInformation-TelephoneNumbers:
   zib deviations:
     ContactPoint.system:
       - cardinality: 1..1
-        reason: Although TelecomType and NumberType are optional in the zib, ContactPoint.system is used to identify the TelephoneNumbers concept itself when this datatype is added to a profile. If only a telephone number is given without TelecomType and NumberType, this will fall in the default slice of the profile.
+        reason: Although TelecomType and NumberType are optional in the zib, ContactPoint.system is used to identify the TelephoneNumbers concept itself when this datatype is added to a profile. If only a telephone number is given without TelecomType and NumberType, this will fall in the default slice of the profile.7
```
Comment (ArdonToonstra) [inline known-issues.yml:76]: This 7 does not belong here.

File: known-issues.yml [hunk 2]
```diff
@@ -21,6 +21,12 @@ pattern-ZibHealthProfessionalReference:
     StructureDefinition:
       - message: "sd-pg-08: 'The title of the StructureDefinition should conform to the profiling guidelines'"
         reason: The title is shown in the hosting profiles as the name of the datatype. "zib HealthProfessional Reference" is more descriptive than the title according to the profiling guidelines ("pattern ZibHealthProfessionalReference").
+        
+zib-AbilityToUseToilet:
+  zib deviations:
+      Observation:
+      - short: AbilityToUseToilet
+        reason: The root concept name is an mistake in the zib and will be fixed at the next pre-publication
```
Comment (ArdonToonstra) [inline known-issues.yml:29]: I was not directly sure what you meant here. Can you include the wrong name 'ToiletUseAbility'  here too?

File: release-notes.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 This document contains release notes per zib, indicating differences with their [STU3 versions](https://simplifier.net/packages/nictiz.fhir.nl.stu3.zib2017/), deviations from the [profiling guidelines](https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4) and other points of interest.
 
+## zib-AbilityToUseToilet
+* No specific changes have been made to this profile other than the generic changes for the Observation resource
```
Comment (ArdonToonstra) [inline release-notes.md:6]: misses a dot at the end.

**Comment by ArdonToonstra** [comment]
@Ashwin-nictiz  -- I added a generated example which now produces two errors. I do however think the errors are incorrect and are caused because the validator does not have access to the Dutch part of the snomed codes... Could you investigate and handle this with for example a known-issue orso?

**Comment by LuudSlagter** [comment]
Validation errors w.r.t. example are not valid, as the code is present in the Dutch extension of SNOMED (checked by performing a local QA).

**Review by LuudSlagter — APPROVED** [review]

**Pull request merged**